PWGCF/Tasks/CMakeLists.txt

```diff
@@ -29,3 +29,7 @@ o2physics_add_dpl_workflow(correlations
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::PWGCFCore
                     COMPONENT_NAME Analysis)
 
+o2physics_add_dpl_workflow(flow-generic-framework
+                    SOURCES GenericFramework.cxx
```

Comment:
Start with lower case.
Call the file like the executable --> flowGenericFramework.cxx